src/DSA/LRU/LRU_test.go

```diff
@@ -0,0 +1,46 @@
+package lru
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLRU(t *testing.T) {
+	lru := NewLRU[string, int](3)
+
+	result := lru.Get("foo")
+	assert.Nil(t, result, "Expected: nil, got: %v", result)
```

Comment:
`lru.Get` doesn't return nillable value, but if we'll return *V then it'll be mess in tests with a lot of value dereferencing.
That's why I return additional boolean that means is it nil or not (like in Queue https://github.com/nacknime-official/kata-machine-go/blob/612588caece6ca6296017813cc704b9b4b7f470a/src/DSA/Queue/Queue.go#L15)